charts/latest/azuredisk-csi-driver/templates/csi-azuredisk-node.yaml

```diff
@@ -15,6 +15,7 @@ spec:
         app: csi-azuredisk-node
     spec:
       hostNetwork: true
+      serviceAccountName: csi-azuredisk-node-sa
```

Comment:
controller should also have access to the secret

Reply:
has bound the role and service account of the controller in rbac-csi-azuredisk-secret.yaml